docs/source/guides/getting_started/index.rst

```diff
@@ -17,34 +16,105 @@ If you plan on using a :ref:`Docker headless image variant
+metadata for those media, and how to place the resulting files into your media library,
+and more.
 
-Must-know terminology:
+Presets and subscriptions accept overrides
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- ``subscription``: URL(s) that you want to download with specific metadata requirements.
-- ``preset``: A media profile comprised of YAML configuration that can specify anything from metadata layout, media quality, or any feature of ytdl-sub, to apply to subscriptions. A preset can inherit other presets.
-- ``prebuilt preset``: Presets that are included in ytdl-sub. These do most of the work defining plugins, overrides, etc in order to make downloads ready for player consumption.
-- ``override``: Verb describing the act of overriding something in a preset. For example, the TV Show presets practically expect you to *override* the URL variable to tell ytdl-sub where to download from.
-- ``override variables``: User-defined variables that are intended to *override* something.
-- ``subscription file``: The file to specify all of your subscriptions and some override variables.
+Presets accept override keys and values and the preset uses those overrides to modify
+their plugin configurations. Similarly, individual subscriptions can supply overrides of
+their presets for just that subscription.
 
-Intermediate terminology:
+Subscriptions are grouped by indentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- ``plugin``: Modular logic to apply to a subscription. To use a plugin, it must be defined in a preset.
-- ``config file``: An optional file where you can define custom presets and other advanced configuration.
-- ``yt-dlp``: The underlying application that handles downloading for ytdl-sub.
+Most subscriptions have more in common with each other than not. Thus, defining the presets
+and overrides for each subscription would result in mostly repetition and would multiply
+the burden of management for the user. The more subscriptions the more work.
 
-Advanced terminology:
+To avoid this redundant work, and so that the subscription configurations describe the
+intent of the user, subscriptions are nested/indented under parent/ancestor keys that
+define their shared configuration. To support this, ``ytdl-sub`` uses special handling
+of the ancestor YAML keys above each subscription. A subscription is the most
+nested/indented/descendant key that specifies the URLs for that subscription. The
+ancestor keys above that subscription describe the shared presets of that subscription
+and all the other descendant subscriptions under them.
+
+Genres are also more often shared between subscriptions than not. To accommodate that
+reality, the ancestor keys of subscriptions may also use the special ``~`` prefix to
+specify genres that also apply to all descendant subscriptions.
+
+Finally, ancestor keys may use the `` | `` special character to combine multiple presets
```

Comment:
since we've introduced inheritance at this point, worth noting that the right-most values of `|` overwrite the former.